src/pages/explorer/index.page.tsx

```diff
@@ -6,30 +6,57 @@ import {
   InputLeftElement,
   Select,
   Text,
-} from "@chakra-ui/react";
-import type { NextPage } from "next";
-import React, { useEffect, useMemo, useRef, useState } from "react";
-import { Layout } from "@/components/Layout";
-import { useQuery } from "@apollo/client";
+} from '@chakra-ui/react';
+import type { NextPage } from 'next';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import { Layout } from '@/components/Layout';
+import { useQuery } from '@apollo/client';
 import {
   GET_ALL_NOTES,
   GET_TAGS_AND_ENTITIES,
-} from "@/services/Apollo/Queries";
-import { SearchIcon } from "@chakra-ui/icons";
-import { useSpring, a } from "react-spring";
-import Fuse from "fuse.js";
-import { filterUniqueObjects } from "@/common/utils";
-import Router from "next/router";
-import { bodyText } from "@/theme";
-import * as styles from "./styles";
+} from '@/services/Apollo/Queries';
+import { SearchIcon } from '@chakra-ui/icons';
+import { useSpring, a } from 'react-spring';
+import Fuse from 'fuse.js';
+import { filterUniqueObjects } from '@/common/utils';
+import Router from 'next/router';
+import { bodyText } from '@/theme';
+import * as styles from './styles';
+import { SearchOS, SearchTypes } from '@chainverse/os';
```

Comment:
I don't see `SearchTypes` as a comonent in the `chainverse-os` repo: https://github.com/DiamondDAO/chainverse-os/tree/main/src/components

Should I be looking somewhere else?

Reply:
<img width="652" alt="image" src="https://user-images.githubusercontent.com/108299975/180300230-f98212a4-19e9-4313-8cbc-7027b7b66a4d.png">
@clemp searchType its not a component. as the name its a Type